### PR TITLE
[17.0][MIG] product_multi_image: Migration to 17.0

### PR DESCRIPTION
**Pull request by psugne**
Note, as OCA v16.0 don't have migrated product_multi_image yet, formatted patch 17.0-versada branch.

### CODE REVIEW

**Comment by psugne** [comment]
Wondering how to deal with test where `odoo-addon-base_multi_image` package is not found, as it's migrated only to our [repo](https://github.com/versada/server-tools/tree/17.0-versada/base_multi_image) for now.

**Comment by oerp-odoo** [comment]
> Wondering how to deal with test where `odoo-addon-base_multi_image` package is not found, as it's migrated only to our [repo](https://github.com/versada/server-tools/tree/17.0-versada/base_multi_image) for now.

So if its migrted to target branch, why it is failing? You should update your src branch with target, to test on expected changes.

File: product_multi_image/tests/test_product_multi_image.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3).
+
+from odoo.tests import common
+
+from .. import hooks
+
+
+class TestProductMultiImage(common.TransactionCase):
+    def setUp(self):
```
Comment (oerp-odoo) [inline product_multi_image/tests/test_product_multi_image.py]: Should use `setUpClass`, to make setup faster.

**Comment by psugne** [comment]
> > Wondering how to deal with test where `odoo-addon-base_multi_image` package is not found, as it's migrated only to our [repo](https://github.com/versada/server-tools/tree/17.0-versada/base_multi_image) for now.
> 
> So if its migrted to target branch, why it is failing? You should update your src branch with target, to test on expected changes.

@oerp-odoo oh, I got it. I checked out from 17.0 branch to be able create PR to OCA too.

**Comment by oerp-odoo** [comment]
> > > Wondering how to deal with test where `odoo-addon-base_multi_image` package is not found, as it's migrated only to our [repo](https://github.com/versada/server-tools/tree/17.0-versada/base_multi_image) for now.
> > 
> > 
> > So if its migrted to target branch, why it is failing? You should update your src branch with target, to test on expected changes.
> 
> @oerp-odoo oh, I got it. I checked out from 17.0 branch to be able create PR to OCA too.

So for oca, that other module should be migrated first. Otherwise its gonna fail on their side as well:)

**Comment by psugne** [comment]
> > > > Wondering how to deal with test where `odoo-addon-base_multi_image` package is not found, as it's migrated only to our [repo](https://github.com/versada/server-tools/tree/17.0-versada/base_multi_image) for now.
> > > 
> > > 
> > > So if its migrted to target branch, why it is failing? You should update your src branch with target, to test on expected changes.
> > 
> > 
> > @oerp-odoo oh, I got it. I checked out from 17.0 branch to be able create PR to OCA too.
> 
> So for oca, that other module should be migrated first. Otherwise its gonna fail on their side as well:)

@oerp-odoo I know. From OCA everything is moving at a snail's pace :snail:

**Review by oerp-odoo — APPROVED** [review]

**Pull request merged**